rollup.config.js

```diff
@@ -57,106 +57,103 @@ const outputOptions = [{
   onwarn,
 }];
 
-if (NODE_ENV === 'production') {
-  plugins.unshift(
-    cleanup({
-      // Ideally we'd also clean third_party/three, which saves
-      // ~45kb in filesize alone... but takes 2 minutes to build
-      include: ['lib/**'],
-      comments: 'none',
-    }),
-  );
+if (NODE_ENV !== 'development') {
```

Comment:
The default run mode for the build script was set as "development", but should probably be "production" so that naively running Rollup with the config builds all the things.